addons/account/models/chart_template.py

```diff
@@ -218,6 +218,31 @@ def _create_demo_data(self):
             # Do not rollback installation of CoA if demo data failed
             _logger.exception('Error while loading accounting demo data')
 
+    def _get_extra_modules(self):
+        loc_module = self.get_xml_id()[self.id].split('.')[0][5:]
+        extra_modules = [
+            module
+            for module, countries in {
+                'account_check_printing': ['us', 'ca'],
+                'account_edi_proxy_client': ['it'],
+                'base_vat': [
+                    'at', 'be', 'ca', 'co', 'de_skr03', 'de_skr04', 'ec', 'es', 'et', 'fr', 'gr',
+                    'it', 'lu', 'mx', 'nl', 'no', 'pl', 'pt', 'ro', 'si', 'tr', 'gb', 've', 'vn',
+                    'syscohada',
+                ],
+                'l10n_gcc_invoice': ['sa'],
+            }.items()
+            if loc_module in countries
```

Comment:
Because? Same complexity because you are iterating once on the whole list in both case (once to build it or once to check if the value is in it). But the set is taking more memory.

Reply:
loc_module in countries is O(n) if you use a list ; I'm not sure to get your point. Of course, n is quite small here in most cases, but still.

Reply:
And `set(countries)` is `O(n)` too.

Why do you want to use sets?

Reply:
Actually, if we want to optimize, I should use a `tuple(*tuple(str, tuple(str)))` instead of a `dict(str, list(str))` like I'm doing here.

Reply:
> And `set(countries)` is `O(n)` too. Same for `{'l10n_xx', 'l10n_yy', ...}`
> 
> Why do you want to use sets?

I'm not telling you to do set(countries). Why not declaring {'us', 'ca'} instead of ['us', 'ca'], directly in the dict, for example ?
A set is O(1) once you do the 'in', and has unique elements, that's all. I'm not saying this is critical in any way. 


> Actually, if we want to optimize, I should use a tuple(*tuple(str, tuple(str))) instead of a dict(str, list(str)) like I'm doing here.

I was going to say the same thing. But frankly, I think it's just "des optimisations de bouts de chandelles".

Reply:
> A set is O(1) once you do the 'in', and has unique elements, that's all.

But you also need to build the set.

> I think it's just "des optimisations de bouts de chandelles"

Exactly.